src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -10,10 +10,27 @@
     <CodeAnalysisRuleSet>..\..\FakeItEasy.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <PropertyGroup Label="Package properties">
+    <Title>FakeItEasy - Mocking made faking easy!</Title>
+    <Authors>Patrik Hägne, FakeItEasy contributors</Authors>
+    <Description>It's faking amazing! The easy mocking framework for .NET that works great in C# and VB.NET alike. No need to know the difference between a stub, a mock or a spy, everything's a fake! The easy to use, refactoring friendly API makes faking a breeze.</Description>
+    <Copyright>Copyright (c) FakeItEasy contributors. (fakeiteasyfx@gmail.com)</Copyright>
+    <PackageProjectUrl>http://fakeiteasy.github.io/</PackageProjectUrl>
```

Comment:
Weird. I don't see that.

![image](https://user-images.githubusercontent.com/1864281/27362695-7326cb62-5630-11e7-83b3-d2515918cedd.png)